test/test.ts

```diff
@@ -1,16 +1,34 @@
-import { VhdlLinter } from '../lib/vhdl-linter';
+import { OIDiagnostic, VhdlLinter } from '../lib/vhdl-linter';
 import { cwd } from 'process';
 import { readdirSync, readFileSync, lstatSync } from 'fs';
 import { join } from 'path';
 import { ProjectParser } from '../lib/project-parser';
 import { } from 'vscode';
+import { OIRange } from '../lib/parser/objects';
 function readDirPath(path: string) {
   return readdirSync(path).map(file => join(path, file));
 }
 interface MessageWrapper {
   file: string,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  messages: any
+  messages:( OIDiagnostic | { message: string})[]
+}
+function isOIDiagnostic(obj: unknown): obj is OIDiagnostic {
+  if ((obj as OIDiagnostic)?.range instanceof OIRange) {
+    return true;
+  }
+  return false;
+}
+function prettyPrintMessages(messages: MessageWrapper[]) {
+  return messages.map(message => {
+    return `file: ${message.file.replace(cwd(), '')}\n` +
+      message.messages.map((innerMessage) => {
+    if (isOIDiagnostic(innerMessage)) {
+      return `${innerMessage.range.start.line}:${innerMessage.range.start.character} - ${innerMessage.range.end.line}:${innerMessage.range.end.character}: ${innerMessage.message}`;
+    }
+    return innerMessage.message;
+  }).join('\n');
+  }).join('\n---------\n');
```

Comment:
```suggestion
function getMessageColor(message: OIDiagnostic | { message: string}) {
  if (isOIDiagnostic(message) && message.severity === DiagnosticSeverity.Error) {
    return '\u001b[31m';
  } else if (isOIDiagnostic(message) && message.severity === DiagnosticSeverity.Warning) {
    return '\u001b[33m';
  }
  return '\u001b[34m';
}
function prettyPrintMessages(messages: MessageWrapper[]) {
  return messages.map(message => {
    const filename = message.file.replace(cwd(), '');
    return message.messages.map((innerMessage) => {
      const messageText = `${getMessageColor(innerMessage)}${innerMessage.message}\u001b[0m`;
      if (isOIDiagnostic(innerMessage)) {
        return `${filename}:${innerMessage.range.start.line + 1}\n  ${messageText}`; // lines are 0 based in OI
      }
      return `${filename}\n  ${messageText}`;
    }).join('\n');
  }).join('\n');
}
```

This prints it even prettier (you can alt+click the file name + line number to directly jump to it and errors are red, etc.).

For example:
![image](https://user-images.githubusercontent.com/10208785/197587368-c873aeb7-90b7-4aa6-81f2-0200afd681d4.png)

Reply:
Okay, changed. But i re-added the complete unmodified range for debugging